src/main/java/org/openelisglobal/patienttype/daoimpl/PatientTypeDAOImpl.java

```diff
@@ -224,8 +224,7 @@ public PatientType readPatientType(String idString) {
     public List<PatientType> getPatientTypes(String description) throws LIMSRuntimeException {
         List<PatientType> list = new Vector<>();
         try {
-            String sql = "from patientType l where upper(l.description) like upper(:param) order by"
-                    + " upper(l.description)";
```

Comment:
also disacrd tis formating change here

Reply:
i dont know what exactly you want me to do coz that was a chenge i made in the dao so that it can filter the patienttypes by string  it cause before that it wascreating an error 
do you want me to revert the changes? @mozzy11

Reply:
Theres no any logical change here. Its just a Formating change.
disacrd it.